intellij/org.eclipse.xtext.xtext.idea/src/org/eclipse/xtext/idea/wizard/XtextModuleBuilder.xtend

```diff
@@ -153,6 +153,7 @@ class XtextModuleBuilder extends ModuleBuilder {
 	}
 
 	def setupWizardConfiguration(WizardConfiguration wizardConfiguration) {
```

Comment:
explicit return type void may help to avoid future signature changes as seen in the compiled code